index.html

```diff
@@ -3612,11 +3527,12 @@ <h3>Usage Patterns</h3>
 person.
           </li>
           <li>
-When the same subject identifier of a credential refers to the same subject 
-across presentations or verifiers, then loss of privacy is possible. Even 
-when different credentials are presented, if the subject identifier is the 
-same, verifiers (and those with access to verifier logs) could infer that
-the holder of the credential is the same person.
+The same <a>subject</a> identifier of a <a>credential</a> refers to the same
```

Comment:
You have missed out "When" from the beginning of this sentence.

Reply:
Thanks. Fixed.